pom.xml

```diff
@@ -76,6 +76,7 @@
     <project.build.outputEncoding>UTF-8</project.build.outputEncoding>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <maven.findbugs.failure.strict>true</maven.findbugs.failure.strict>
+    <maven-release-plugin.version>2.5.3</maven-release-plugin.version>
```

Comment:
It is always the same debate of "developers confort" vs "reproducibility/rules/.."

```
mvn release:prepare release:perform -Darguments=-Dmaven-compiler-plugin.version=x.y
```

You can always do many crappy stuffs even without this with `-Darguments=` :-)

I'm always supporting all efforts to enforce the quality of builds but it mustn't be again the maintainability and the developer efficiency. The root issue is that we don't publish the flat pom when we do a release ...

For me having a corporate/organisational pom to add some additional configurations/conventions is the way to go and it must be as possible extensible/configurable without being too verbose.

But I agree with you that we must follow as possible the default properties and settings which are known by maven users and plugins. For example in our parent pom ( https://github.com/jenkinsci/pom/blob/master/pom.xml#L586 ) hardcoding the java version at the compiler plugin level is a bad thing. We should override the default well known properties `maven.compiler.target` and `maven.compiler.source`.